save-plugins/fix-plugin/README.md

```diff
@@ -46,13 +46,14 @@ suiteName = "DocsCheck"
 execFlags="-F"
 testFilePattern="*Test.kt"
 expectedFilePattern="*Expected.kt"
-batchSize = 1
+batchSize = 1 # (default value)
+separator = ", " # (default value)
```

Comment:
```suggestion
batchSeparator = ", " # (default value, not used if `batchSize = 1`)
```
As these two settings are connected, they should have similar names

Reply:
Fixed